src/main/java/uk/gov/pay/ledger/transaction/model/TransactionFactory.java

```diff
@@ -137,6 +148,38 @@ private Transaction createPayment(TransactionEntity entity) {
         return null;
     }
 
+    private Exemption createExemption(String exemption3ds, String exemption3dsRequested) {
+        Exemption exemption = null;
+        if(exemption3ds != null) {
+            boolean requested = Exemption3ds.from(exemption3ds) != EXEMPTION_NOT_REQUESTED;
+            String type = null;
+           if (Exemption3dsRequested.from(exemption3dsRequested) == CORPORATE) {
+                type = CORPORATE.toString();
+            }
```

Comment:
Unlike the connector version of this, it looks like this will add a `"type": "corporate"` even if the `Exemption3ds` is `EXEMPTION_NOT_REQUESTED`. This does not make sense semantically and should Never Happen™️ but can we make it behave the same as connector so that ledger and connector will at least return consistent things if such bad data does ever occur?